docs/resources/shared-infrastructure.md

```diff
@@ -8,6 +8,8 @@ Some of these are useful for our development workflows, while others are used as
 We have [a GitHub organization](https://github.com/jupyterhub/) for hosting all of our code repositories.
 This organization is where we do most of the code-related work for the project, and where we have discussions and coordination.
 
+(resources:shared-drive)=
+
 ## Google Drive
 
 We have a [**JupyterHub Team Google Drive**](https://drive.google.com/drive/u/1/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8) to store documents, presentations, images, etc that are useful to the team.
```

Comment:
```suggestion
We have a [**JupyterHub Team Google Drive**](https://drive.google.com/drive/u/0/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8) to store documents, presentations, images, etc that are useful to the team.
```

The `/u/1/` in the Google drive URL means access wll be requested from the users's second Google account if they're logged in to multiple accounts. This should probably be `/u/0/`?

Reply:
oops good point - I replaced it with the `shared link` which is user-agnostic